src/locales/locale-en.json

```diff
@@ -328,5 +329,12 @@
   "write a message": "Write a message...",
   "write the first message": "Write the first message...",
   "set store location": "Set location of this store",
-  "set group location": "Set location of this group"
+  "set group location": "Set location of this group",
+  "STORESTATUS":{
+    "ACTIVE":"in cooperation",
```

Comment:
In general everything starts with an uppercase letter, is there a reason to do it differently here?